sycl/test/lit.cfg.py

```diff
@@ -142,6 +143,9 @@
 
 acc_run_substitute=f"env SYCL_DEVICE_FILTER={filter} "
 if xocc != "off":
+    # cleanup the named semaphore in case the previous test didnt cleanup properly
+    # If someone tries to run multiple tests on the same machin this could cause issues.
```

Comment:
```suggestion
    # Clean up the named semaphore in case the previous test did not clean up properly.
    # If someone tries to run multiple tests on the same machine this could cause issues.
```